org.splash.Splash.json

```diff
@@ -0,0 +1,103 @@
+{
+    "app-id": "org.splash.Splash",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "splash",
+    "rename-desktop-file": "splash.desktop",
+    "rename-icon": "splash",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--device=all",
+        "--filesystem=host"
+    ],
+    "cleanup": [
+        "/bin/gsl_*",
+        "/bin/opencv_*",
+        "/include",
+        "/lib/debug",
+        "/share/OpenCV",
+        "/share/info",
+        "/share/man",
+        "*.a",
+        "*.la",
+        "*.cmake"
+    ],
+    "modules": [
+        {
+            "name": "opencv",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/opencv/opencv.git",
+                    "tag": "3.4.3"
+                }
+            ]
+        },
+        {
+            "name": "portaudio",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.assembla.com/portaudio.git",
+                    "tag": "pa_stable_v190600_20161030"
+                }
+            ]
+        },
+        {
+            "name": "gsl",
+            "config-opts": [
+                "--enable-shared",
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://mirror.sergal.org/gnu/gsl/gsl-2.5.tar.gz",
+                    "sha256": "0460ad7c2542caaddc6729762952d345374784100223995eb14d614861f2258d"
+                }
+            ]
+        },
+        {
+            "name": "jsoncpp",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/open-source-parsers/jsoncpp",
+                    "tag": "1.8.4"
+                }
+            ]
+        },
+        {
+            "name": "splash",
+            "buildsystem": "simple",
+            "config-opts": [
+                "-DBUILD_GENERIC_ARCH=ON"
+            ],
+            "build-commands": [
+                "./make_deps.sh",
+                "cmake -DBUILD_GENERIC_ARCH=ON -DCMAKE_INSTALL_PREFIX=/app .",
+                "make -j$(nproc) && make install",
+                "install -Dm644 org.splash.Splash.appdata.xml /app/share/appdata/org.splash.Splash.appdata.xml"
```

Comment:
New metainfo files should use .metainfo.xml as an extension instead of .appdata.xml and installed under $prefix/share/metainfo